### DISCUSSION TITLE
Move a project to xarray-contrib

### DISCUSSION THREAD

**Discussion by stefraynaud** [pydata/xarray#5167] [Q&A] [answered]
Hi there,

I have a project that is heavily based on xarray (https://github.com/VACUMM/xoa/).
I would like to move it to the xarray-contrib organisation to reach a more appropriate audience.

Is it eligible? Who decides? And How to proceed?

Have a nice day.

**Comment by max-sixty** [comment]
@pydata/xarray thoughts? Do we have a policy? We can discuss on the call tomorrow.

**Reply by keewis** [reply]
I don't think we have a policy (besides not being too specific), but I agree, we should discuss this tomorrow.

**Comment by keewis** [comment, marked as answer]
we decided to document the process in [xarray-contrib/xarray-contrib](https://github.com/xarray-contrib/xarray-contrib/blob/main/README.md). In short: open a issue on that repo to request the migration.
